### ISSUE TITLE
[nomer append globi-correct] on single lowercase long text phrase takes ~1 minute

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/nomer#182]
to reproduce:

```
time echo -e "\t7 small acorns or other seed found in cheek pouches during preparation. Seeds are brown, pointed at one end and flattened on the other, 0.75 x 0.6cm"\
 | nomer append globi-correct
```

yielded 

```
	7 small acorns or other seed found in cheek pouches during preparation. Seeds are brown, pointed at one end and flattened on the other, 0.75 x 0.6cm	SAME_AS		acorns or   found  cheek pouches during preparation   are brown  pointed at  end  flattened  the   x  cm					

real	0m58.756s
user	1m1.295s
sys	0m0.129s
```

**Comment by jhpoelen** [comment]
Root cause appears to be ```StringUtils.capitalize``` of commons-lang v2.6 with the following test failing:

```java
    @Test(timeout = 10)
    public void capitalizeLongName() {
        String aLongName = "acorns or   found  cheek pouches during preparation   are brown  pointed at  end  flattened  the   x  cm";
        String corrected = StringUtils.capitalize(aLongName);
        assertThat(corrected, is(aLongName));
    }
```

**Comment by jhpoelen** [comment]
After implementing a work around by capitalizing only the first sequence of lower case letters, the processing time of [nomer append globi-correct] is reduced to less than a second, including jvm startup time / initialization. 

```
time echo -e "\t7 small acorns or other seed found in cheek pouches during preparation. Seeds are brown, pointed at one end and flattened on the other, 0.75 x 0.6cm"\
 | nomer append globi-correct
```

produced: 

```
[main] INFO org.globalbioticinteractions.nomer.match.ResourceServiceReadOnly - using cached [https://github.com/globalbioticinteractions/globi-taxon-names/raw/main/taxon-name-mapping.csv] at [/home/jorrit/.cache/nomer/hash/sha256/b959e969ddf4114bd590ec1cdcf7ec572076bd46e2e28e2fee038a3f6d41b9fd/3916de5192d63fa8956124a172650f396b843afb11d95b6865530b13968fb87e.gz]
[main] INFO org.globalbioticinteractions.nomer.match.ResourceServiceReadOnly - using cached [https://github.com/globalbioticinteractions/globi-taxon-names/raw/main/non-taxon-words.txt] at [/home/jorrit/.cache/nomer/hash/sha256/b959e969ddf4114bd590ec1cdcf7ec572076bd46e2e28e2fee038a3f6d41b9fd/d6f27c41de517496a1afbe090fe2c31cf08c15bba44b0076b4d2590194be2cb1.gz]
	7 small acorns or other seed found in cheek pouches during preparation. Seeds are brown, pointed at one end and flattened on the other, 0.75 x 0.6cm.	SAME_AS		Acorns or   found  cheek pouches during preparation   are brown  pointed at  end  flattened  the   x  cm					

real	0m0.757s
user	0m1.363s
sys	0m0.136s
```

**Comment by jhpoelen** [comment]
fix/workaround available in  Nomer v0.5.12 https://github.com/globalbioticinteractions/nomer/releases/tag/0.5.12
